packages/orchestrator-ui-components/src/api/axios.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020-2021 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import axios, { AxiosRequestConfig } from 'axios';
+
+const apiPath = 'http://localhost:8080/api/surf/forms';
```

Comment:
Having a localhost url in the code base is not preferred.
Is the form also available on the Dev environment? Or maybe in an Okteto namespace?

Reply:
It's merged now on dev.

Will adapt in #274 : which is the branch that will probably be merged.